### PR TITLE
basebox 18.04 support

### PR DESCRIPTION
**Pull request by giorgiosironi**
@lsh-0 add your own packages too.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Missing `realpath` as [Java 7 has been removed](https://github.com/elifesciences/builder-private/commit/dc3c9695ea15db7a9707e626dce02e504f3514d7)

**Comment by giorgiosironi** [comment]
There are all sort of checks for `16.04` or `trusty` around the various states, I'm limiting myself to what is necessary for `basebox` to be built for now.

File: elife/python.sls [hunk 1]
```diff
@@ -1,4 +1,5 @@
-{% if salt['grains.get']('osrelease') == "16.04" %}
+{% set osrelease = salt['grains.get']('osrelease') %}
+{% if osrelease == "16.04" or osrelease == "18.04" %}
```
Comment (lsh-0) [inline elife/python.sls]: perhaps `{% if osrelease != "14.04" %}` which is a bit more future proof

**Comment by giorgiosironi** [comment]
A Vagrant VM with `bento/ubuntu-18.04` builds correctly, but this build will run more stuff in heavybox so it will probably still fail.

**Comment by giorgiosironi** [comment]
Ok, this passes, but only because it's built on 14.04 for now. Renaming to `basebox support` and merging as a first step.

**Pull request merged**